presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/PruneUnreferencedOutputs.java

```diff
@@ -358,7 +358,7 @@ public PlanNode visitAggregation(AggregationNode node, RewriteContext<Set<Variab
                 if (context.get().contains(variable)) {
                     Aggregation aggregation = entry.getValue();
                     expectedInputs.addAll(extractUniqueVariables(aggregation, symbolAllocator.getTypes()));
-                    aggregation.getMask().ifPresent(expectedInputs::add);
+                    aggregation.getMask().ifPresent(mask -> expectedInputs.add(mask));
```

Comment:
no need to change

Reply:
Could be a result of rebase (Symbol/Variable conversion is now removed).